tutorials/multicore/mtbb101_fillNtuples.C

```diff
@@ -40,7 +40,7 @@ Int_t mtbb101_fillNtuples()
 
    // Create a random generator and and Ntuple to hold the numbers
    TRandom3 rndm(1);
-   TFile ofile("mp101_singleCore.root", "RECREATE");
+   TFile ofile("mpbb101_singleCore.root", "RECREATE");
```

Comment:
```suggestion
   TFile ofile("mtbb101_singleCore.root", "RECREATE");
```
since the script name has a 't'?